checkov/serverless/checks/function/aws/AdminPolicyDocument.py

```diff
@@ -18,7 +18,7 @@ def scan_function_conf(self, conf):
         :return: <CheckResult>
         """
         key = IAM_ROLE_STATEMENTS_TOKEN
-        if key in conf.keys():
+        if conf.get(key):
             for statement in conf[key]:
```

Comment:
```suggestion
        statements = conf.get(IAM_ROLE_STATEMENTS_TOKEN)
        if statements and isinstance(statements, list):
            for statement in statements:
```
also line 20 is obsolete, but couldn't select it 😄